README.md

```diff
@@ -132,6 +132,9 @@ MARKDOWN=false
 # Once you've created a client ID and secret in your Google project use the below environment variables to enable SSO
 # GOOGLE_CLIENT_ID=abc123
 # GOOGLE_CLIENT_SECRET=123abc
+# When running in docker, you may face issues with the protocol or URL that is redirecting the call, override that by specifying the full redirect URL in the below variables
+# GOOGLE_SIGNIN_REDIRECT=https://wishlist.example.com/auth/google/redirect
```

Comment:
I'd prefer to have the user set the base URL of the site (similar to the ROOT_PATH we have now) and extrapolate from there. There's some complexity with that, but changing how variables like this work in the future is painful because it's a breaking change.

Maybe a ROOT_URL variable that is required if specifying GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET?

So I could do something like ROOT_URL=https://wishlist.example.com/subpath and it would automatically overwrite base with its path? So something like this:
```ts
function appendSlash(path: string) {
  if (path.endsWith('/')) return path
  return path + '/'
}

...
rootUrl: appendSlash(process.env.ROOT_URL ?? process.env.ROOT_PATH ?? '/')
...

// The base path is used in HTML templates rather than the fully qualified path, mostly for legacy reasons. It also has the following advantages:
//  * It saves a negligible amount of bandwidth.
//  * It allows the admin to deploy to multiple subdomains. This isn't really supported, but it's nice if the admin migrates domains on the same path (typically /), and can keep the old one around because users won't migrate immediately.
// This used to be set with ROOT_PATH, but is now calculated based on ROOT_URL, which falls back to the former if unset.
// rootUrl may be a fully-qualified `https://domain/path/`-style value, or may simply be `/`, or a subpath.
try {
  config.base = new URL(config.rootUrl).pathname
} catch {
  config.base = config.rootUrl
}
```

Reply:
@Wingysam, do you want to implement this last piece? Sounds like you have it in your head what you're looking for.

Reply:
Sure, I'll do that part!